app/src/main/java/com/yalta/services/AddPointService.kt

```diff
@@ -0,0 +1,14 @@
+package com.yalta.services
+
+import com.yalta.repositories.PointRepo
+import com.yalta.repositories.process
+
+class AddPointService(private val repo: PointRepo) {
```

Comment:
why new service? oh, cause they are created in ctor...
ok...

Reply:
Mybe just pass repo then?

Reply:
discussed